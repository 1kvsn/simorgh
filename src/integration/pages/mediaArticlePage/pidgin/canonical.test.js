/**
 * @service pidgin
 * @pathname /pidgin/articles/cvpde7nqj92o
 */

import runCanonicalTests from '../canonicalTests';

describe('Canonical', () => {
  describe(pageType, () => {
    runCanonicalTests(service);
  });
});
