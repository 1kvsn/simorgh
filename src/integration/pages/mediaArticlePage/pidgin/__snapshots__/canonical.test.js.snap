// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical Media Article Page A11y I can see the skip to content link 1`] = `"Waka go wetin de inside"`;

exports[`Canonical Media Article Page Analytics ATI 1`] = `"<img height=\\"1px\\" width=\\"1px\\" alt=\\"\\" style=\\"position:absolute\\" src=\\"https://logws1363.ati-host.net?s=598343&amp;s2=70&amp;p=pidgin.articles.cvpde7nqj92o.page&amp;x1=[urn%3Abbc%3Aoptimo%3Aasset%3Acvpde7nqj92o]&amp;x2=[responsive]&amp;x3=[news-pidgin]&amp;x4=[pcm]&amp;x7=[article-sfv]&amp;x8=[simorgh-nojs]&amp;x9=[WS%2520Media%2520First%2520page%2520(1)]&amp;x11=[2023-01-17T14%3A03%3A06.410Z]&amp;x12=[2023-01-17T14%3A15%3A57.894Z]\\"/>"`;

exports[`Canonical Media Article Page Footer Anchors should match text and url 1`] = `
Object {
  "text": "BBC News, Pidgin",
  "url": "/pidgin",
}
`;

exports[`Canonical Media Article Page Footer Anchors should match text and url 2`] = `
Object {
  "text": "Why you fit trust BBC News",
  "url": "https://www.bbc.com/pidgin/institutional-48528766",
}
`;

exports[`Canonical Media Article Page Footer Anchors should match text and url 3`] = `
Object {
  "text": "How dem dey take use am",
  "url": "https://www.bbc.co.uk/usingthebbc/terms/",
}
`;

exports[`Canonical Media Article Page Footer Anchors should match text and url 4`] = `
Object {
  "text": "As e concern BBC",
  "url": "https://www.bbc.co.uk/aboutthebbc",
}
`;

exports[`Canonical Media Article Page Footer Anchors should match text and url 5`] = `
Object {
  "text": "Privacy Policy",
  "url": "https://www.bbc.co.uk/usingthebbc/privacy/",
}
`;

exports[`Canonical Media Article Page Footer Anchors should match text and url 6`] = `
Object {
  "text": "Cookies",
  "url": "https://www.bbc.co.uk/usingthebbc/cookies/",
}
`;

exports[`Canonical Media Article Page Footer Anchors should match text and url 7`] = `
Object {
  "text": "Call BBC",
  "url": "https://www.bbc.co.uk/pidgin/send/u50853577",
}
`;

exports[`Canonical Media Article Page Footer Anchors should match text and url 8`] = `
Object {
  "text": "De way wey we de take go external link.",
  "url": "https://www.bbc.co.uk/editorialguidelines/guidance/feeds-and-links",
}
`;

exports[`Canonical Media Article Page Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/pidgin"`;

exports[`Canonical Media Article Page Footer I can see the footer copyright and external linking text 1`] = `"Â© 2023 BBC. De external site no concern BBC. De way wey we de take go external link."`;

exports[`Canonical Media Article Page Header I can see the branding 1`] = `"BBC News, Pidgin"`;

exports[`Canonical Media Article Page Header Navigation link should match text and url 1`] = `
Object {
  "text": "Home",
  "url": "/pidgin",
}
`;

exports[`Canonical Media Article Page Header Navigation link should match text and url 2`] = `
Object {
  "text": "Nigeria",
  "url": "/pidgin/topics/c2dwqd1zr92t",
}
`;

exports[`Canonical Media Article Page Header Navigation link should match text and url 3`] = `
Object {
  "text": "Africa",
  "url": "/pidgin/topics/c404v061z85t",
}
`;

exports[`Canonical Media Article Page Header Navigation link should match text and url 4`] = `
Object {
  "text": "World",
  "url": "/pidgin/topics/c0823e52dd0t",
}
`;

exports[`Canonical Media Article Page Header Navigation link should match text and url 5`] = `
Object {
  "text": "Video",
  "url": "/pidgin/topics/c3l19z3k1ert",
}
`;

exports[`Canonical Media Article Page Header Navigation link should match text and url 6`] = `
Object {
  "text": "Sport",
  "url": "/pidgin/topics/cjgn7gv77vrt",
}
`;

exports[`Canonical Media Article Page Header Navigation link should match text and url 7`] = `
Object {
  "text": "Entertainment",
  "url": "/pidgin/topics/cqywjyzk2vyt",
}
`;

exports[`Canonical Media Article Page Header Navigation link should match text and url 8`] = `
Object {
  "text": "Most popular",
  "url": "/pidgin/popular/read",
}
`;

exports[`Canonical Media Article Page Main heading should match text 1`] = `"WS Media (1) -Media above title"`;

exports[`Canonical Media Article Page SEO Apple Touch Icon should match attributes 1`] = `
Object {
  "sizes": null,
  "url": "http://localhost:7080/pidgin/images/icons/icon-192x192.png",
}
`;

exports[`Canonical Media Article Page SEO Apple Touch Icon should match attributes 2`] = `
Object {
  "sizes": "72x72",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/pidgin/images/icons/icon-72x72.png",
}
`;

exports[`Canonical Media Article Page SEO Apple Touch Icon should match attributes 3`] = `
Object {
  "sizes": "96x96",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/pidgin/images/icons/icon-96x96.png",
}
`;

exports[`Canonical Media Article Page SEO Apple Touch Icon should match attributes 4`] = `
Object {
  "sizes": "128x128",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/pidgin/images/icons/icon-128x128.png",
}
`;

exports[`Canonical Media Article Page SEO Apple Touch Icon should match attributes 5`] = `
Object {
  "sizes": "144x144",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/pidgin/images/icons/icon-144x144.png",
}
`;

exports[`Canonical Media Article Page SEO Apple Touch Icon should match attributes 6`] = `
Object {
  "sizes": "152x152",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/pidgin/images/icons/icon-152x152.png",
}
`;

exports[`Canonical Media Article Page SEO Apple Touch Icon should match attributes 7`] = `
Object {
  "sizes": "180x180",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/pidgin/images/icons/icon-180x180.png",
}
`;

exports[`Canonical Media Article Page SEO Apple Touch Icon should match attributes 8`] = `
Object {
  "sizes": "192x192",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/pidgin/images/icons/icon-192x192.png",
}
`;

exports[`Canonical Media Article Page SEO Apple Touch Icon should match attributes 9`] = `
Object {
  "sizes": "384x384",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/pidgin/images/icons/icon-384x384.png",
}
`;

exports[`Canonical Media Article Page SEO Apple Touch Icon should match attributes 10`] = `
Object {
  "sizes": "512x512",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/pidgin/images/icons/icon-512x512.png",
}
`;

exports[`Canonical Media Article Page SEO Canonical link 1`] = `"http://localhost:7080/pidgin/articles/cvpde7nqj92o"`;

exports[`Canonical Media Article Page SEO Dir attribute 1`] = `"ltr"`;

exports[`Canonical Media Article Page SEO FB app ID 1`] = `"1609039196070050"`;

exports[`Canonical Media Article Page SEO Heading level 1 1`] = `"WS Media (1) -Media above title"`;

exports[`Canonical Media Article Page SEO Lang attribute 1`] = `"pcm"`;

exports[`Canonical Media Article Page SEO Linked data should match text 1`] = `
Object {
  "@context": "http://schema.org",
  "@graph": Array [
    Object {
      "@type": "NewsArticle",
      "author": Object {
        "@type": "NewsMediaOrganization",
        "logo": Object {
          "@type": "ImageObject",
          "height": 576,
          "url": "https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png",
          "width": 1024,
        },
        "name": "BBC News Pidgin",
        "noBylinesPolicy": "https://www.bbc.com/pidgin/institutional-48528766#authorexpertise",
      },
      "dateModified": "2023-01-17T14:15:57.894Z",
      "datePublished": "2023-01-17T14:03:06.410Z",
      "headline": "WS Media First page (1)",
      "image": Object {
        "@type": "ImageObject",
        "height": 576,
        "url": "https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png",
        "width": 1024,
      },
      "inLanguage": Object {
        "@type": "Language",
        "alternateName": "pcm",
        "name": "Nigerian Pidgin",
      },
      "mainEntityOfPage": Object {
        "@id": "http://localhost:7080/pidgin/articles/cvpde7nqj92o",
        "@type": "WebPage",
        "name": "WS Media First page (1)",
      },
      "publisher": Object {
        "@type": "NewsMediaOrganization",
        "logo": Object {
          "@type": "ImageObject",
          "height": 576,
          "url": "https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png",
          "width": 1024,
        },
        "name": "BBC News Pidgin",
        "publishingPrinciples": "https://www.bbc.com/pidgin/institutional-48528766",
      },
      "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png",
      "url": "http://localhost:7080/pidgin/articles/cvpde7nqj92o",
    },
  ],
}
`;

exports[`Canonical Media Article Page SEO Linked data should match text 2`] = `
Object {
  "@context": "http://schema.org",
  "@type": "VideoObject",
  "description": "BBC launch trailer for We Know Our Place women's sport campaign\\"",
  "duration": "PT54S",
  "embedURL": "https://www.test.bbc.com/ws/av-embeds/articles/cvpde7nqj92o/p01thw22/pcm",
  "name": "BBC launch trailer for We Know Our Place women's sport campaign",
  "thumbnailUrl": "https://ichef.test.bbci.co.uk/images/ic/1024x576/p01thw3g.jpg",
  "uploadDate": "2022-07-01T08:56:56.000Z",
}
`;

exports[`Canonical Media Article Page SEO OG description 1`] = `"WS Media First page (1)"`;

exports[`Canonical Media Article Page SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png"`;

exports[`Canonical Media Article Page SEO OG image alt 1`] = `"BBC News Pidgin"`;

exports[`Canonical Media Article Page SEO OG locale 1`] = `"pcm"`;

exports[`Canonical Media Article Page SEO OG site name 1`] = `"BBC News Pidgin"`;

exports[`Canonical Media Article Page SEO OG title 1`] = `"WS Media First page (1) - BBC News Pidgin"`;

exports[`Canonical Media Article Page SEO OG type 1`] = `"article"`;

exports[`Canonical Media Article Page SEO OG url 1`] = `"http://localhost:7080/pidgin/articles/cvpde7nqj92o"`;

exports[`Canonical Media Article Page SEO Page title 1`] = `"WS Media First page (1) - BBC News Pidgin"`;

exports[`Canonical Media Article Page SEO Robots meta tag 1`] = `"noodp, noydir, max-image-preview:large"`;

exports[`Canonical Media Article Page SEO Twitter card 1`] = `"summary_large_image"`;

exports[`Canonical Media Article Page SEO Twitter creator 1`] = `"@BBCNews"`;

exports[`Canonical Media Article Page SEO Twitter description 1`] = `"WS Media First page (1)"`;

exports[`Canonical Media Article Page SEO Twitter image alt 1`] = `"BBC News Pidgin"`;

exports[`Canonical Media Article Page SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png"`;

exports[`Canonical Media Article Page SEO Twitter site 1`] = `"@BBCNews"`;

exports[`Canonical Media Article Page SEO Twitter title 1`] = `"WS Media First page (1) - BBC News Pidgin"`;

exports[`Canonical Media Article Page Timestamp should match text and date 1`] = `
Object {
  "date": "PT54S",
  "text": "00:54",
}
`;
